samples/bluetooth/bap_broadcast_sink/src/main.c

```diff
@@ -281,83 +289,115 @@ static bool subgroup_get_valid_bis_indexes_cb(const struct bt_bap_base_subgroup
 							      ? subgroup_chan_allocation
 							      : BT_AUDIO_LOCATION_MONO_AUDIO;
 		}
+		/* keep allocation for later use */
+		bis_index->audio_allocation[data.bis[i].index - 1] = data.bis[i].chan_allocation;
+		printk("bis_index->audio_allocation[%u] = 0x%08x\n", data.bis[i].index - 1,
+		       data.bis[i].chan_allocation);
```

Comment:
Please add a comment for the `- 1` - Also consider storing the `data.bis[i].index - 1` in a local variable with a meaningful name to avoid performing the calculation twice (and with a meaningful name, it may be also improve readability).